Add Vivek Agrawal to Contributors list

access2content opened the pull request.

access2content committed:
Add Vivek Agrawal to Contributors list

Roshanjossey commented:
Hi, I appreciate you cleaning up the format but this results in multiple merge conflicts and prevents me from merging. Could you please submit another pull request by adding your name only?  Cheers

access2content commented:
I am not an expert on Git, so here is what I did.

I deleted the add-vivek-agrawal branch on my fork of firstcontributors that
was 2 months ago.
Then cloned the repository from my account into my system.
Edited the contributors.md file and added just my name.
Pushed this into the repository.
Created a Pull Request.

I was getting an error/warning that said there is a conflict, but you can
still create a request. So I did that.

Let me know if this was correct. If not, what should I do?
Sorry for creating any problems.

Regards,
Vivek Agrawal


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Sat, Sep 19, 2020 at 10:32 PM Roshan Jossy <notifications@github.com>
wrote:

> Hi, I appreciate you cleaning up the format but this results in multiple
> merge conflicts and prevents me from merging. Could you please submit
> another pull request by adding your name only? Cheers
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/firstcontributions/first-contributions/pull/29545#issuecomment-695332384>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKR3LYOBUJZGEKEQPCWKJXTSGTP3DANCNFSM4PWZDK4Q>
> .
>

Roshanjossey commented:
Hi Vivek, yes, that's correct. I've resolved the merge conflict and merged your pull request.

To handle merge conflicts yourself, checkout the following material
https://github.com/firstcontributions/first-contributions/blob/master/additional-material/git_workflow_scenarios/keeping-your-fork-synced-with-this-repository.md
https://github.com/firstcontributions/first-contributions/blob/master/additional-material/git_workflow_scenarios/resolving-merge-conflicts.md
